src/fmu/sumo/utilities/scripts/sim2sumo.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+
+"""Upload data to Sumo from FMU."""
+
+import warnings
+import os
+import argparse
+import logging
+from pathlib import Path
+
+try:
```

Comment:
This may not be an issue anymore, so we may not need this try/catch.

Reply:
Have removed it.